src/lib/Components/DottedLine.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react"
+import { requireNativeComponent, View } from "react-native"
+
+class DottedLine extends React.Component<any, any> {
+  static propTypes = { dotScale: React.PropTypes.number }
```

Comment:
Not sure if emission has updated, but `React.PropTypes` has been deprecated and is available in the `prop-types` package: 

```js
import PropTypes from 'prop-types'
```

Reply:
Ah yes. I'll switch it- thanks.

Reply:
Can you add documentation for the `dotScale` prop?